### PR TITLE
Looping over logs of all jobs using a single command

### PR DESCRIPTION
**Pull request by anokun7**
Using the label constuct `-l` to loop over output of all jobs (or pods) using a single command

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by anokun7** [comment]
I signed it

File: content/en/docs/tasks/job/parallel-processing-expansion.md [hunk 1]
```diff
@@ -119,14 +119,10 @@ process-item-banana-wrsf7   0/1       Completed   0          4m
...
-There is not a single command to check on the output of all jobs at once,
-but looping over all the pods is pretty easy:
+We can use this single command to check on the output of all jobs at once:
 
 ```shell
-for p in $(kubectl get pods -l jobgroup=jobexample -o name)
-do
-  kubectl logs $p
-done
+  kubectl logs -f -l jobgroup=jobexample
```
Comment (sftim) [inline content/en/docs/tasks/job/parallel-processing-expansion.md]:
```suggestion
kubectl logs -f -l jobgroup=jobexample
```

File: content/en/docs/tasks/job/parallel-processing-expansion.md [hunk 2]
```diff
@@ -119,14 +119,10 @@ process-item-banana-wrsf7   0/1       Completed   0          4m
 process-item-cherry-dnfu9   0/1       Completed   0          4m
 ```
 
-There is not a single command to check on the output of all jobs at once,
-but looping over all the pods is pretty easy:
+We can use this single command to check on the output of all jobs at once:
```
Comment (sftim) [inline content/en/docs/tasks/job/parallel-processing-expansion.md]:
```suggestion
For this simple example, we can view the output via logs from the pods:
```
(to emphasize that a different approach is appropriate to read output from production jobs, especially if there are very many of them)

Comment (anokun7) [reply content/en/docs/tasks/job/parallel-processing-expansion.md]: @sftim I prefer the original wording. Thank you very much.

**Pull request merged**